backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglGL20.java

```diff
@@ -595,7 +595,7 @@ public void glStencilOpSeparate (int face, int fail, int zfail, int zpass) {
 	public void glTexImage2D (int target, int level, int internalformat, int width, int height, int border, int format, int type,
 		Buffer pixels) {
 		if (pixels == null)
-			GL11.glTexImage2D(target, level, internalformat, width, height, border, format, type, (ByteBuffer)null);
+			GL11.glTexImage2D(target, level, internalformat, width, height, border, format, type, 0L);

```

Comment:
This change breaks FrameBufferTest for the LWJGL2 backend. It gives:
```
Exception in thread "LWJGL Application" org.lwjgl.opengl.OpenGLException: Cannot use offsets when Pixel Unpack Buffer Object is disabled
	at org.lwjgl.opengl.GLChecks.ensureUnpackPBOenabled(GLChecks.java:125)
	at org.lwjgl.opengl.GL11.glTexImage2D(GL11.java:2899)
	at com.badlogic.gdx.backends.lwjgl.LwjglGL20.glTexImage2D(LwjglGL20.java:598)
	at com.badlogic.gdx.graphics.glutils.GLOnlyTextureData.consumeCustomData(GLOnlyTextureData.java:78)
	at com.badlogic.gdx.graphics.GLTexture.uploadImageData(GLTexture.java:270)
	at com.badlogic.gdx.graphics.GLTexture.uploadImageData(GLTexture.java:257)
	at com.badlogic.gdx.graphics.Texture.load(Texture.java:159)
	at com.badlogic.gdx.graphics.Texture.<init>(Texture.java:147)
	at com.badlogic.gdx.graphics.Texture.<init>(Texture.java:142)
	at com.badlogic.gdx.graphics.glutils.FrameBuffer.createTexture(FrameBuffer.java:81)
	at com.badlogic.gdx.graphics.glutils.FrameBuffer.createTexture(FrameBuffer.java:1)
	at com.badlogic.gdx.graphics.glutils.GLFrameBuffer.build(GLFrameBuffer.java:174)
	at com.badlogic.gdx.graphics.glutils.FrameBuffer.<init>(FrameBuffer.java:75)
	at com.badlogic.gdx.graphics.glutils.FrameBuffer.<init>(FrameBuffer.java:57)
	at com.badlogic.gdx.tests.FrameBufferTest.create(FrameBufferTest.java:132)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:150)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:127)
```
I don't know what the right fix is, or if LWJGL checks are even correct.

Reply:
Reverted it for now since its a major regression. we would need to expose different methods as we are using null value to flag two separate cases which the checks are conflicting with.

Reply:
I apologize guys, i should have test more this core change. I'll PR something soon.